### PR TITLE
node: specify the DNS domain

### PR DESCRIPTION
**Pull request by giuseppe**
Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by sdodson** [comment]
[test]

File: roles/docker/tasks/systemcontainer_crio.yml [hunk 1]
```diff
@@ -134,6 +134,14 @@
     image: "{{ l_crio_image }}"
     state: latest
 
+- name: Remove CRI-o default configuration files
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+  - /etc/cni/net.d/200-loopback.conf
```
Comment (ingvagabund) [inline roles/docker/tasks/systemcontainer_crio.yml]: +2 spaces

File: roles/docker/tasks/systemcontainer_crio.yml [hunk 2]
```diff
@@ -134,6 +134,14 @@
...
+  - /etc/cni/net.d/100-crio-bridge.conf
```
Comment (ingvagabund) [inline roles/docker/tasks/systemcontainer_crio.yml]: +2 spaces

**Comment by ingvagabund** [comment]
The install job failed due to a flake. @giuseppe can you squash the commits? Once done it can be merged.

**Pull request merged**